docs/src/main/asciidoc/tika-guide.adoc

```diff
@@ -126,3 +94,141 @@ public class TikaParserResource {
+
+    @Inject
+    TikaParser parser;
+
+    @POST
+    @Path("/text")
+    @Consumes({"application/pdf", "application/vnd.oasis.opendocument.text"})
+    @Produces(MediaType.TEXT_PLAIN)
+    public String extractText(InputStream stream) {
+        Instant start = Instant.now();
+
+        String text = parser.getText(stream);
+
+        Instant finish = Instant.now();
+
+        log.info(Duration.between(start, finish).toMillis() + " mls have passed");
+
+        return text;
+    }
+}
+----
+
+This basic JAX-RS resource has a resource method which accepts PDF and OpenDocument format documents and uses an injected `TikaParser` to parse them and report the extracted text. It also measures how long does it take to parse a given document.
+
+== Run the application
+
+Now we are ready to run our application. Use:
+
+[source,shell]
+----
+./mvnw compile quarkus:dev
+----
+
+and you should see output similar to:
+
+.quarkus:dev Output
+[source,shell]
+----
+$ mvn clean compile quarkus:dev
+[INFO] Scanning for projects...
+[INFO]
+INFO] --------------------< org.acme.example:apache-tika >--------------------
+[INFO] Building apache-tika 1.0-SNAPSHOT
+[INFO] --------------------------------[ jar ]---------------------------------
+...
+Listening for transport dt_socket at address: 5005
+2019-10-15 14:23:26,442 INFO  [io.qua.dep.QuarkusAugmentor] (main) Beginning quarkus augmentation
+2019-10-15 14:23:26,960 INFO  [io.qua.resteasy] (build-15) Resteasy running without servlet container.
+2019-10-15 14:23:26,960 INFO  [io.qua.resteasy] (build-15) - Add quarkus-undertow to run Resteasy within a servlet container
+2019-10-15 14:23:26,991 INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 549ms
+2019-10-15 14:23:27,637 INFO  [io.quarkus] (main) Quarkus 999-SNAPSHOT started in 1.361s. Listening on: http://0.0.0.0:8080
+2019-10-15 14:23:27,638 INFO  [io.quarkus] (main) Profile dev activated. Live Coding activated.
+2019-10-15 14:23:27,639 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy, tika]----
+
+Now that the REST endpoint is running, we can get it to parse PDF and OpenDocument documents using a command line tool like curl:
+
+[source,shell]
+----
+$ curl -X POST -H "Content-type: application/pdf" --data-binary @target/classes/quarkus.pdf http://localhost:8080/parse/text
+Hello Quarkus
+----
+
+and
+
+[source,shell]
+----
+$ curl -X POST -H "Content-type: Content-type: application/vnd.oasis.opendocument.text" --data-binary @target/classes/quarkus.odt http://localhost:8080/parse/text
+Hello Quarkus
+----
+
+== Explore the Solution
+
+The solution repository located in the `apache-tika` {quickstarts-archive-url}[directory] contains the examples we have worked through in this quickstart guide.
+We suggest that you check out the quickstart solutions and explore the `apache-tika` directory to learn more about the {extension-name} extension features.
```

Comment:
And you should get rid of that part once you have a structure similar to the other guides.